myapp/spec/models/task_spec.rb

```diff
@@ -27,4 +27,30 @@
       ).with_prefix
     }
   end
+
+  describe 'scope' do
+    describe 'sort_by_keyword' do
+      let!(:first_task) { create(:task, name: 'ううう') }
+      let!(:second_task) { create(:task, name: 'あああ') }
+      let!(:third_task) { create(:task, name: 'いいい') }
+
+      subject { Task.sort_by_keyword(sort) }
+
+      context 'when sort type is id_asc' do
+        let(:sort) { 'id_asc' }
+
+        it 'sort by specified sort type' do
+          is_expected.to eq [first_task, second_task, third_task]
+        end
+      end
+
+      context 'when sort type is id_desc' do
+        let(:sort) { 'id_desc' }
+
+        it 'sort by specified sort type' do
+          is_expected.to eq [third_task, second_task, first_task]
+        end
+      end
+    end
+  end
```

Comment:
scopeは実質クラスメソッドなので今回の場合だと`Task.sort_by_keyword`で表せるので、`scope`ブロックは無くてもいいんじゃないな〜と思ってます！

> 以下でもわかるように、スコープでのメソッドの設定は、クラスメソッドの定義と完全に同じ (というよりクラスメソッドの定義そのもの) です。どちらの形式を使用するかは好みの問題です。
引用: [Railsガイド](https://railsguides.jp/active_record_querying.html#%E3%82%B9%E3%82%B3%E3%83%BC%E3%83%97)

```suggestion
  describe '.sort_by_keyword' do
    let!(:first_task) { create(:task, name: 'ううう') }
    let!(:second_task) { create(:task, name: 'あああ') }
    let!(:third_task) { create(:task, name: 'いいい') }

    subject { Task.sort_by_keyword(sort) }

    context 'when sort type is id_asc' do
      let(:sort) { 'id_asc' }

      it 'sort by specified sort type' do
        is_expected.to eq [first_task, second_task, third_task]
      end
    end

    context 'when sort type is id_desc' do
      let(:sort) { 'id_desc' }

      it 'sort by specified sort type' do
        is_expected.to eq [third_task, second_task, first_task]
      end
    end
  end
```